reconcile/prometheus_rules_tester.py

```diff
@@ -66,8 +70,8 @@ def get_prometheus_rules(cluster_name):
         rules[r["path"]] = {}
 
     for n in gqlapi.query(orb.NAMESPACES_QUERY)["namespaces"]:
-        cluster = n["name"]
-        namespace = n["cluster"]["name"]
+        namespace = n["name"]
+        cluster = n["cluster"]["name"]
```

Comment:
I know this doesn't belong to this, but messages are quite confusing and `--cluster-name` option doesn't currently work. I can move it to a different PR if reviewers prefer.